src/main/modules/nunjucks/index.ts

```diff
@@ -101,6 +103,10 @@ export class Nunjucks {
     nunjucksEnv.addGlobal('UnemploymentType', UnemploymentType)
     nunjucksEnv.addGlobal('BankAccountType', BankAccountType)
     nunjucksEnv.addGlobal('ClaimStatus', ClaimStatus)
+    nunjucksEnv.addGlobal('AppPaths', AppPaths)
+    if (FeatureToggles.isEnabled('finePrint')) {
+      nunjucksEnv.addGlobal('cookieText', `GOV.UK uses cookies make the site simpler. <a href="${AppPaths.cookiesPage.uri}">Find out more about cookies</a>`)
```

Comment:
Where is this global used? Also, will this text be translated?

Reply:
in layout.njk add t() at line 9: 
```<p>{{ t(cookieText) | safe }}</p>```

Reply:
Thanks for spotting this!

It is the seen_cookie_messag - This will be a dropdown pop up when a user goes to our service for the first time, or it is the first time they view this in the last month.